### PR TITLE
test removing some lines

### PR DESCRIPTION
**Pull request by pld**

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
-      - "/home/runner/work/fhircore/fhircore/android/engine"
+    - ^/home/runner/work/fhircore/fhircore/android/engine.*
   geowidget:
     paths:
-      - "/home/runner/work/fhircore/fhircore/android/geowidget"
+    - ^/home/runner/work/fhircore/fhircore/android/geowidget.*
   quest:
     paths:
-      - "/home/runner/work/fhircore/fhircore/android/quest"
+    - ^/home/runner/work/fhircore/fhircore/android/quest.*
```
Comment (ndegwamartin) [inline codecov.yml:37]: Should these path definitions be starting with a caret?

Comment (pld) [reply codecov.yml:37]: yea maybe not, I copied this directly from here, which was the last valid run, https://app.codecov.io/github/opensrp/fhircore/commit/5182221f688cfb24ae13720bb50221c2e040107e

File: codecov.yml [hunk 2]
```diff
@@ -1,34 +1,39 @@
 codecov:
-  disable_default_path_fixes: yes
+  disable_default_path_fixes: true
```
Comment (ndegwamartin) [inline codecov.yml:2]: Also since we've disabled the default path fixes we probably need to specify our paths manually

Comment (pld) [reply codecov.yml:2]: let me try removing this line, this has always been disabled, but worth a try

**Comment by pld** [comment]
yay this hash leads to processes the coverage reports successfully, https://github.com/opensrp/fhircore/pull/2118/commits/d803c43d5f5a33e3cf6eba29b8cfd1d6aa01b3aa, remaining problems are
- status is being reported for each flag, we don't care about per flag status

**Pull request merged**